src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2017 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.featurelayers.feature_layer_geopackage;
+
+import java.io.File;
+import java.util.List;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.data.GeoPackage;
+import com.esri.arcgisruntime.data.GeoPackageFeatureTable;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class FeatureLayerGeoPackageSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Feature Layer GeoPackage Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map with the streets vector basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
+
+      // create a GeoPackage from a local gpkg file
+      File geoPackageFile = new File("./samples-data/auroraCO/AuroraCO.gpkg");
```

Comment:
A more modern way would be to do something like `Paths.get("./samples-data/auroraCO/AuroraCO.gpkg").toAbsolutePath()`.